holoviews/core/data/dictionary.py

```diff
@@ -270,6 +270,13 @@ def values(cls, dataset, dim, expanded=True, flat=True, compute=True, keep_index
         return values
 
 
+    @classmethod
+    def assign(cls, dataset, new_data):
```

Comment:
Interface methods do not currently have docstrings. I'll consistently add those at some point but since they aren't really public we've not prioritized that yet.